pkgs/by-name/me/mealie/package.nix

```diff
@@ -1,25 +1,38 @@
-{ lib
-, stdenv
-, callPackage
-, fetchFromGitHub
-, makeWrapper
-, nixosTests
-, python3Packages
-, writeShellScript
+{
+  lib,
+  stdenv,
+  callPackage,
+  fetchFromGitHub,
+  makeWrapper,
+  nixosTests,
+  python3Packages,
+  writeShellScript,
 }:
 
 let
-  version = "2.3.0";
+  version = "2.6.0";
   src = fetchFromGitHub {
     owner = "mealie-recipes";
     repo = "mealie";
     rev = "v${version}";
-    hash = "sha256-GN+uXyZCvDuFmQnXhn0mFans3bvvEw7Uq6V0OeCPEbE=";
+    hash = "sha256-txkHCQ/xTakPXXFki161jNOKwAH9p9z1hCNEEkbqQtM=";
   };
 
   frontend = callPackage (import ./mealie-frontend.nix src version) { };
 
-  pythonpkgs = python3Packages;
+  pythonpkgs = python3Packages.override {
+    overrides = _self: super: {
+      openai = super.openai.overridePythonAttrs (_old: rec {
```

Comment:
You could also just cherry-pick that commit, which is a better solution I believe. But feel free to leave it as is if you deem it better for some reason.
https://github.com/mealie-recipes/mealie/commit/3b12a62fc602abf2c7922bd167c9161af3caa07f